tools/docker/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
     env_file:
       - default.env
       - .env
-    entrypoint: [ '/bin/bash', '/var/scripts/docker-db-start-and-autoupgrade.sh' ]
+    entrypoint: [ '/usr/bin/env', 'bash', '/var/scripts/docker-db-start-and-autoupgrade.sh' ]
```

Comment:
If
```suggestion
    entrypoint: [ 'bash', '/var/scripts/docker-db-start-and-autoupgrade.sh' ]
```
didn't work, it would probably have been better to just leave this one. The container is almost certainly going to have it at `/bin/bash` anyway.

Reply:
Confirmed this works.